### PR TITLE
fix build to work and build within container

### PR DESCRIPTION
**Pull request by joshoohaah**

### CODE REVIEW

**Comment by sevans-symplr** [comment]
Really seems like these should be separate PRs. The changes to package-lock.json I'm going to guess are mostly due to a newer version of NPM being used than what was originally checked in.

File: tsconfig.json [hunk 1]
```diff
@@ -4,6 +4,7 @@
         "moduleResolution": "node",
         "sourceMap": false,
         "jsx": "react",
+        "skipLibCheck": true,
```
Comment (sevans-symplr) [inline tsconfig.json:7]: Why was this necessary?

**Review by sevans-symplr — CHANGES_REQUESTED** [review]

File: tsconfig.json (hunk 1 above)
Comment (joshoohaah) [reply tsconfig.json:7]: yeah. this is 100% the reason for this.   the TS extension is causing the issue.  see other issue I opened up against the forked repo.
https://github.com/microsoft/AzureDevOps-WSJF-Extension/issues/16

**Review by sevans-symplr — APPROVED** [review]

File: tsconfig.json (hunk 1 above)
Comment (joshoohaah) [reply tsconfig.json:7]: I would love an option to NOT do this..however I don't see an alternative.
